### ISSUE TITLE
A11Y: Yellow on white (and vice versa) fails colour contrast

### ISSUE THREAD

**Issue by ianmcburnie** [marko-js/markojs-website-v5#97]
## Yellow on White

<img width="736" alt="Screen Shot 2023-05-23 at 12 10 54 PM" src="https://github.com/marko-js/website/assets/38065/d6432f3a-412a-4c51-90b6-4d30768b210c">

## White on Yellow

<img width="736" alt="Screen Shot 2023-05-23 at 12 12 15 PM" src="https://github.com/marko-js/website/assets/38065/a305dd08-22b7-472d-950a-d3e4adcb1b27">

**Comment by ianmcburnie** [comment]
Offline decision was made to change the yellow to an orange. We will need a shade that passes colour contrast for normal sized text, due to the button text in "Code Eliminates" section.

**Comment by ianmcburnie** [comment]
This applies to the green too.

<img width="725" alt="Screen Shot 2023-06-08 at 9 56 52 AM" src="https://github.com/marko-js/website/assets/38065/ecc04f30-ac58-4642-b7b3-6336c0b9e085">
